src/platform-includes/profiling/index/why-profiling/javascript.mdx

```diff
@@ -0,0 +1,5 @@
+## Why profiling?
+
+With [profiling](/product/profiling/), Sentry tracks your software's performance by sampling your program's call stack in a variety of environments, collecting function level information about your code and enables you to fine tune the performance of your program. Sentry profiling captures function calls and their exact locations, aggregates them and shows you the most common code paths of your program, which surfaces areas of optimization opportunity that can help you increase both the performance of your code and increase user satisfaction.
```

Comment:
```suggestion
With [profiling](/product/profiling/), Sentry tracks your software's performance by taking samples from your program's call stack in a variety of environments. Collecting function level information about your code and enabling you to fine tune the performance of your program. Sentry profiling captures function calls and their exact locations, aggregates them and shows you the most common code paths of your program. These profiles then represent the best opportunities for you to optimize your code and make your user's happier.
```

i like the usage of you and your, it reads very well about how I can benefit. 

The end just read off to me, so I suggested a slight change. You can ignore my suggestion there. I wanted to call out referring to them as a "profile", as that feels like a concrete thing now as a user reading the docs, instead of a sum of samples.